lib/ronin/recon/builtin/net/cert_sh.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+#
+# ronin-recon - A micro-framework and tool for performing reconnaissance.
+#
+# Copyright (c) 2023 Hal Brodigan (postmodern.mod3@gmail.com)
+#
+# ronin-recon is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ronin-recon is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with ronin-recon.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+require 'ronin/recon/worker'
+
+require 'async/http/internet/instance'
+
+module Ronin
+  module Recon
+    module Net
+      #
+      # A recon worker that grabs the SSL/TLS certificate from open ports that
+      # use SSL/TLS.
+      #
+      class CertSh < Worker
+
+        register 'net/cert_sh'
+
+        accepts Domain
+
+        summary 'Returns host from each domains certificate.'
+
+        description <<~DESC
+          Returns host from each domains certificate.
```

Comment:
Probably should mention that it's querying `cert.sh` in the description and summary.